libs/dh/shared/data-access-mocks/src/lib/marketParticipant.ts

```diff
@@ -354,3 +357,9 @@ function getAuditLogByOrganizationId() {
     return res(ctx.delay(300), ctx.data({ __typename: 'Query', organizationAuditLog: auditLog }));
   });
 }
+
+function getAuditLogByActorId() {
+  return mockGetAuditLogByActorIdQuery((req, res, ctx) => {
+    return res(ctx.data(getActorAuditLogsMock));
```

Comment:
suggestion: let's add some dalay, similar to other mocks


```suggestion
    return res(ctx.delay(300), ctx.data(getActorAuditLogsMock));
```